internal/services/nginx/nginx_deployment_data_source.go

```diff
@@ -164,6 +165,11 @@ func (m DeploymentDataSource) Attributes() map[string]*pluginsdk.Schema {
 			},
 		},
 
+		"automatic_upgrade_channel": {
```

Comment:
We have `auto_scale_profile` and `automatic_upgrade_channel` should we align on one of auto or automatic?

Reply:
resolving this based on offline discussion. We are keeping `automatic_upgrade_channel`.